### PR TITLE
Adds the feature manifest

### PR DESCRIPTION
**Pull request by adborden**
- Adds the feature manifest for feature space.
- Enables legal search on `feature`, `dev`, and `stage`.

### CODE REVIEW

File: manifest_feature.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+inherit: manifest_base.yml
+host: fec-feature-cms
+services:
+  - cms-creds-feature
+  - fec-creds-feature
+  - fec-feature-cms
+env:
+  FEC_API_URL: https://fec-feature-api.18f.gov
+  FEC_CMS_DEBUG: true
+  FEC_CMS_ENVIRONMENT: dev
```
Comment (ccostino) [inline manifest_feature.yml]: This should be `feature`, right?

Comment (adborden) [inline manifest_feature.yml]: Hmm, well, I decided that feature isn't really new environment, it's just another dev environment deployed to a new space. I thought `dev` is more correct here. This setting is only used for the development banner that displays on the site in dev and production

screenshot from 2016-05-12 13-27-27
Displaying FEATURE here doesn't seem right. Folks (like the FEC team) have learned the difference between dev, stage, and production, and calling this a dev environment feels more correct than feature, IMHO.

Comment (ccostino) [inline manifest_feature.yml]: Ah, I didn't realize that's all this is in reference to - thanks for clarifying!

**Comment by ccostino** [comment]
Thanks again, @adborden!

**Pull request merged**